.github/workflows/dev_on_schedule_run_e2e_android.yml

```
@@ -2,8 +2,8 @@ name: '[Nightly] Run E2E tests on Android'
 
 on:
   schedule:
-    # 11:00 PM Paris time in UTC (Monday to Friday)
-    - cron: '0 23 * * 1-5'
+    # 5:00 AM Paris time in UTC (Monday to Friday)
+    - cron: '0 5 * * 1-5'
```

```suggestion
    - cron: '0 6 * * 1-5'
```
S'ils prennent pas trop de temps à tourner, chaud de les décaler encore d'une heure